docs/static_landing_page/css/landing_page_style.css

```diff
@@ -395,7 +395,6 @@ li.check-icon {
 }
 .benefit-card {
 	border-radius: 1px;
-	box-shadow: 0px 40px 80px rgba(0, 0, 0, 0.05);
```

Comment:
If you zoom into our home page right now, near the GitHub button in the navbar you will see a box with a shadow.

**Image for reference**
![image](https://user-images.githubusercontent.com/50991099/158840934-faeb783a-93a3-4aba-8a69-bf4c347c398b.png)

This box is created by the line 398. We don't really need it. 

Additionally, in the collapsed / mobile version it creates the same effect but more pronounced as seen here:

![image](https://user-images.githubusercontent.com/50991099/158841554-e8c6b7cc-5407-4a44-8d6a-7f4a38fd9ad8.png)

In our PR since we are using multiple logos, the box shadow creates a rectangular shadow around them which although subtle creates a striped effect in the collapsed navbar.